Description: Ignore deps in vendor dir/ This patch can be removed once we devendor image/integration
Author: Nilesh Patra <nilesh@debian.org>
Forwarded: not-needed
Last-Update: 2024-05-01
--- a/setup.py
+++ b/setup.py
@@ -1136,7 +1136,7 @@
     go = shutil.which('go')
     if not go:
         raise SystemExit('The go tool was not found on this system. Install Go')
-    required_go_version = subprocess.check_output([go] + 'list -f {{.GoVersion}} -m'.split(), env=dict(os.environ, GO111MODULE="on")).decode().strip()
+    required_go_version = subprocess.check_output([go] + 'list -f {{.GoVersion}} -m -mod=readonly'.split(), env=dict(os.environ, GO111MODULE="on")).decode().strip()
     current_go_version = subprocess.check_output([go, 'version']).decode().strip().split()[2][2:]
     if parse_go_version(required_go_version) > parse_go_version(current_go_version):
         raise SystemExit(f'The version of go on this system ({current_go_version}) is too old. go >= {required_go_version} is needed')
